README.md

```diff
@@ -1,9 +1,54 @@
-# scrappy
+# Scrappy the Slack Bot
 
 ![build](https://github.com/hackclub/scrappy/workflows/build/badge.svg)
 
-the slack bot that powers [scrapbook.hackclub.com](https://scrapbook.hackclub.com). to get started developing, ask @sampoder to be added to the `scrappy (dev)` app on Slack and for a `.env` file. then use `ngrok` to forward your local instance onto the world wide web.
+Scrappy is the Slack bot that powers [scrapbook.hackclub.com](https://scrapbook.hackclub.com). Scrappy automatically generates your Scrapbook and Scrapbook posts via Slack messages. For more information about how to sign up,
```

Comment:
`For more information about how to sign up,` <- this sentence probably needs finishing

Reply:
LOL oops [f709dbd](https://github.com/hackclub/scrappy/pull/317/commits/f709dbda96bc28ffe42a84d9500f169fddf8db27)